components/azure.key.manager/src/test/java/org/wso2/azure/client/AzureAdValues.java

```diff
@@ -0,0 +1,10 @@
+package org.wso2.azure.client;
+
+// https://graph.microsoft.com
+// https://login.microsoftonline.com/025464ed-e523-4fb2-94fe-166720e9f0a2/federationmetadata/2007-06/federationmetadata.xml
```

Comment:
Shall we remove these comments if not required

Reply:
files removed since its not the best solution to test